original/v1/s006-lists/lists.html

```diff
@@ -369,9 +369,13 @@ <h3 class="text-muted" id="show-all">PyLadies Brno</h3>
                             ale nebude to ten samý seznam:
                             měnit se bude nezávisle na tom starém.
                         </div>
+                        <div>
+                            Jinou možností se stejným výsledkem je použití
+                            metody <code>copy</code>.
+                        </div>
```

Comment:
Myslel jsem, že by to někomu mohlo lépe v mysli zapadnout k těm mnoha dalším metodám, které lze na slovníku provádět. Ten `list()` je z tohoto pohledu jaksi výjimečný. Ale na druhou stranu znají `int()`, `str()` a další, takže novinka to taky není. Těžko posoudit, co s nepopsanou myslí a doposavadními znalostmi lépe utkví v paměti. Proto mě napadlo slespoň to stručně zmínit.

Když tenhle commit nebude zakomponovaný, zlobit se nebudu. Díky za ty předešlé. 👍 

PS: Případně že by se rozšířil tahák o `copy`...?